src/travis2docker/templates/Dockerfile

```diff
@@ -62,9 +63,9 @@ WORKDIR ${TRAVIS_BUILD_DIR}
 
 {% if runs -%}
 {% if image == 'quay.io/travisci/travis-python' -%}
-RUN /bin/bash -c "source $HOME/virtualenv/python2.7_with_system_site_packages/bin/activate && source /rvm_env.sh && {{ ' && '.join(runs) }}"
+RUN /bin/bash -c "source $HOME/virtualenv/python{{ python_version }}_with_system_site_packages/bin/activate && source /rvm_env.sh && {{ ' && '.join(runs) }}"
 {% elif  image == 'vauxoo/odoo-80-image-shippable-auto' -%}
-RUN /bin/bash -c "source ${REPO_REQUIREMENTS}/virtualenv/python2.7/bin/activate && source ${REPO_REQUIREMENTS}/virtualenv/nodejs/bin/activate && source /rvm_env.sh && {{ ' && '.join(runs) }}"
+RUN /bin/bash -c "source ${REPO_REQUIREMENTS}/virtualenv/python{{ python_version }}/bin/activate && source ${REPO_REQUIREMENTS}/virtualenv/nodejs/bin/activate && source /rvm_env.sh && {{ ' && '.join(runs) }}"
```

Comment:
Add this virtualenv folder from big image, please

Reply:
@moylop260 
In this other one PR https://github.com/Vauxoo/docker-odoo-image/pull/269 the folder was created